trunk/SUAVE/Methods/Aerodynamics/AVL/translate_data.py

```diff
@@ -218,7 +218,7 @@ def translate_results_to_conditions(cases,results):
         res.aerodynamics.roll_moment_coefficient[i][0]      = case_res.aerodynamics.roll_moment_coefficient
         res.aerodynamics.pitch_moment_coefficient[i][0]     = case_res.aerodynamics.pitch_moment_coefficient
         res.aerodynamics.yaw_moment_coefficient[i][0]       = case_res.aerodynamics.yaw_moment_coefficient
-        res.aerodynamics.lift_coefficient[i][0]             = case_res.aerodynamics.total_lift_coefficient
+        res.aerodynamics.lift_coefficient[i][0]             = case_res.aerodynamics.pitch_moment_coefficient
```

Comment:
Huh?

Reply:
This was probably done by accident when I was shifting around things. I corrected it. Issue resolved.